data-prepper-plugins/otel-trace-raw-processor/src/main/java/com/amazon/dataprepper/plugins/processor/oteltrace/OTelTraceRawProcessor.java

```diff
@@ -74,15 +76,15 @@ public OTelTraceRawProcessor(final PluginSetting pluginSetting) {
                                         .build().toJson()));
                             } catch (Exception ex) {
                                 log.error("Unable to process invalid Span {}:", sp, ex);
-                                spanCounter.increment();
+                                spanErrorsCounter.increment();
                             }
                         }
                     }
                 } catch (Exception ex) {
                     log.error("Unable to process invalid ResourceSpan {} :", rs, ex);
-                    resourceSpanCounter.increment();
+                    resourceSpanErrorsCounter.increment();
                 }
-
+                totalProcessingErrors = spanErrorsCounter.count() + resourceSpanErrorsCounter.count();
```

Comment:
Is this just a private double? How will this be seen by end users?

Reply:
Updating this to be a Counter metric